Makefile

```diff
@@ -14,7 +14,13 @@ logs: up
 
 dependencies:
 	pip install --upgrade pip
+	pip install -r requirements.txt
 	pip install -r tests/requirements.txt
 
-test: dependencies up
+test: functional static_analysis
+
+functional: dependencies up
 	pytest --noconftest tests/functional
+
+static_analysis: dependencies
+	flake8
```

Comment:
Could we wait for #290 to get merged so we can use tox to run `flake8`?

Reply:
We can or we can merge this one and do the requested change after.